inspec.yml

```diff
@@ -11,3 +11,6 @@ summary: '"This Security Technical Implementation Guide is published as a tool
   to this document should be sent via email to the following address:
   disa.stig_spt@mail.mil."'
 version: 2.6.0
+
+# administrator-approved extension IDs
```

Comment:
Did you want to make these inputs?

aka

```yaml
input:
  - name: my_input
    type: string
    required: false
    description: "my human desc of what this is for"
    value: "my defualt value"
 ```
documented here: https://docs.chef.io/inspec/inputs/

Reply:
I can make them, I just wanted to know if I was on the right path.